packages/favorites/src/FavoriteHeart.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { HeartOutlined, HeartFilled, Spinner } from './Icons';
+import css from './FavoriteHeart.module.scss';
+import { useFavorites } from './FavoritesContext';
+
+export const FavoriteHeart = ({
+  id,
+  name,
+  onChange,
+  onMouseDown,
+}: {
+  id: string;
+  name: string;
+  onChange?: (isFavorited: boolean, event: React.ChangeEvent<HTMLInputElement>) => void;
+  onMouseDown?: (event: React.MouseEvent<HTMLInputElement, MouseEvent>) => void;
+}): JSX.Element => {
+  const { isFavorited, isDisabled, isActiveMutation, toggleFavorite } = useFavorites(id);
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (onChange) onChange(isFavorited, e);
+    toggleFavorite();
+  };
+
+  return (
+    <>
```

Comment:
Looks like you don't _technically_ need a fragment here. Did you have it there for some reason?

Reply:
Not anymore. It was there for another little testing div that has been removed. The fragment is also now removed.